test/patch/parse.js

```diff
@@ -248,8 +248,8 @@ Index: test2
     });
     it('should throw on invalid input in strict mode', function() {
       expect(function() {
-        parsePatch('Index: foo\nfoo', {strict: true});
-      }).to['throw'](/Unknown line 2 "foo"/);
+        parsePatch('Index: foo\n+++ bar\nblah', {strict: true});
+      }).to['throw'](/Unknown line 3 "blah"/);
```

Comment:
Ok, I missed the point. I've added a test to check `applyPatches()` work with diffs without `Index` metadata, using `oldFileName` and `newFileName` fields, that's just my use case.